### ISSUE TITLE
Virtualmin Configuration: Spam filtering options - This should be in the server template

### ISSUE THREAD

**Issue by shoulders** [virtualmin/virtualmin-gpl#817]
|  SYSTEM INFORMATION||
|----------------------|---------------------------|
|  OS type and version |     Ubuntu Linux 22.04.4     |
|  Webmin version      |     2.111      |
|  Usermin version     |     2.010     |
|  Virtualmin version  |     7.10.0       |
|  Theme version       |     21.10      |
|  Package updates     |      26  package updates are available   |

## The issue

`Virtualmin --> System Settings --> Virtualmin Configuration --> Configuration category: Spam filtering options`

![image](https://github.com/virtualmin/virtualmin-gpl/assets/319997/8843d138-8ff4-4b53-814a-1439a7359ec3)

This updates `/etc/webmin/virtual-server/config` but does not seem to apply these settings globally. I am not sure when they are read or used, perhaps only when a Virtual Server is created.

## proposed solution

move these configurations to the server template where they are much more logical.

**Comment by jcameron** [comment]
Yeah, these should probably be moved to a template. But for now, they just set the defaults for new domains - this don't update existing domains.

**Comment by iliaross** [comment]
Reopened, as this was closed by an automated cleanup run. @shoulders, sorry for the noise, and thanks for the contribution. We definitely didn't mean any disrespect.

If it turns out the issue is already resolved for you, you're welcome to close it.
